homeassistant/components/cloud/google_config.py

```diff
@@ -105,7 +105,7 @@ def should_2fa(self, state):
         except ErrorResponse as err:
             _LOGGER.warning("Error reporting state - %s: %s", err.code, err.message)
 
-    async def _async_request_sync_devices(self):
+    async def _async_request_sync_devices(self, agent_user_id: str):
```

Comment:
This shouldn't be necessary as we already can access it as `self.agent_user_id` ?